### PR TITLE
Profile Picture Upload

### PR DESCRIPTION
**Pull request by joelklabo**
Video [HERE (YouTube)](https://youtube.com/shorts/v_bG1NMtQQw?feature=share) too long for GitHub upload

Todo:
- [x] localization
- [x] merge #847 or something like it so camera uploads work

**Create account flow**
| Start | Choose | Uploading | Finished |
| - | - | - | - |
| Simulator Screen Shot - iPhone 14 Pro Max - 2023-03-31 at 17 07 15 | Simulator Screen Shot - iPhone 14 Pro Max - 2023-03-31 at 17 07 17 | Simulator Screen Shot - iPhone 14 Pro Max - 2023-03-31 at 17 07 26 | Simulator Screen Shot - iPhone 14 Pro Max - 2023-03-31 at 17 07 32 |

**Edit profile flow**
| Start | Choose |
| - | - |
| Simulator Screen Shot - iPhone 14 Pro - 2023-03-31 at 17 06 32 | Simulator Screen Shot - iPhone 14 Pro - 2023-03-31 at 17 06 28 |

### CODE REVIEW

File: damus/Info.plist [hunk 1]
```diff
@@ -46,9 +46,5 @@
 		<key>NSAllowsArbitraryLoads</key>
 		<true/>
 	</dict>
-	<key>NSCameraUsageDescription</key>
-	<string>Damus needs access to your camera if you want to upload photos from it</string>
-	<key>NSMicrophoneUsageDescription</key>
-	<string>Damus needs access to your microphone if you want to upload recorded videos from it</string>
```
Comment (joelklabo) [inline damus/Info.plist:52]: I'm not sure what's going on with this but it should be in the .plist somewhere

**Comment by jb55** [comment]
yay!

**Comment by jb55** [comment]
looks like there is a conflict between #847 and this PR, perhaps you can just pull that one into here?

**Comment by joelklabo** [comment]
Ok, rebased this one 🤙

**Pull request closed without merging**